packages/js/product-editor/src/components/section/editor.scss

```diff
@@ -23,8 +28,11 @@
 
     &__description {
         margin-top: $gap-small;
-        margin-bottom: 32px;
         font-size: 13px;
         color: $gray-700;
```

Comment:
This one looks like it may be `#1E1E1E` (gray 900) in the current designs.

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/37805/commits/aed6675ea178c3df423999f2e4d139670661d12f